app/javascript/components/tree-view/hierarchical-tree-view.jsx

```diff
@@ -1,72 +1,164 @@
+/* eslint camelcase: ["warn", {allow: ["bs_tree", "tree_name", "click_url", "check_url", "allow_reselect", "hierarchical_check", "silent_activate", "select_node"]}] */
 import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Tree, {
   Node,
-  defaultStore,
-  callBack,
-  ActionTypes,
 } from '@manageiq/react-ui-components/dist/wooden-tree';
+import { http } from '../../http_api';
+import { combineReducers } from '../../helpers/redux';
+import reducers, { ACTIONS } from './reducers/index';
+import basicStore from './reducers/basicStore';
+import './style.scss';
```

Comment:
@himdel is this okay? Do we want to load the styling like this?

Reply:
Well, we're already importing some css that way, I'm OK with that.

(I'm kinda waiting to see what webpack 5 does for non-js modules, because I'm all for doing it one consistent way, but I also really like the idea of having a single `import "specific_component/"` that loads everything necessary for that component.)

Reply:
Yeah, nevermind ... we're moving this to the global space because there's already some tree css there. We can revisit this after the left side trees are all sorted out.